src/System.Private.CoreLib/shared/System/Guid.cs

```diff
@@ -123,8 +120,7 @@ public Guid(int a, short b, short c, byte d, byte e, byte f, byte g, byte h, byt
         private enum GuidParseThrowStyle : byte
         {
             None = 0,
-            All = 1,
-            AllButOverflow = 2
```

Comment:
This code is not dead. Check the test failures.

Reply:
Right.  For example, note https://github.com/dotnet/coreclr/blob/fcc025a5456d09ce1960cc4ec331a89d9a67e03a/src/System.Private.CoreLib/shared/System/Guid.cs#L157 determining what exception to throw based on this.